spec/factories/games.rb

```diff
@@ -13,5 +13,50 @@
       white_player_id 1
       black_player_id 2
     end
+
+    trait :check_scenario do
```

Comment:
@danielpowell4 Thanks! Good suggestions - I've added tests for vertical, diagonal, and knight checks